Card/src/main/java/com/paypal/android/card/Card.kt

```diff
@@ -1,7 +1,7 @@
 package com.paypal.android.card
 
 data class Card(
-    val cardNumber: String,
-    val expirationDate: String,
-    val securityCode: String
+    var number: String = "",
+    var expirationDate: String = "",
+    var securityCode: String = ""
```

Comment:
We can I remember David saying `Card` has a ton of optional fields. I know on the BT side we sometimes skip security codes. It might be good to re-visit in that next ticket for adding card fields.

Reply:
I think we can still keep these fields as values instead of variables and keep them optional in the constructor. As long as there's a default value, a merchant is able to omit that field from the constructor.

```kotlin
data class Card(
    val number: String = "",
    val expirationDate: String = "",
    val securityCode: String = ""
)

val card = Card("1234512345", "02/2022")
```

This would just prevent the modification of each value after the card instance is created.

Reply:
Following up here, Can we convert these vars to vals? Also from looking at the documentation for the confirm call, `number` and `expiration` are required fields. Should we remove the default values here?